graphene/utils/subclass_with_meta.py

```diff
@@ -46,10 +45,10 @@ def __init_subclass__(cls, **meta_options):
                 "Abstract types can only contain the abstract attribute. "
                 "Received: abstract, {option_keys}"
             ).format(option_keys=", ".join(options.keys()))
-        else:
-            super_class = super(cls, cls)
-            if hasattr(super_class, "__init_subclass_with_meta__"):
-                super_class.__init_subclass_with_meta__(**options)
+
+        super_class = super(cls, cls)
+        if hasattr(super_class, "__init_subclass_with_meta__"):
+            super_class.__init_subclass_with_meta__(**options)
```

Comment:
Without this change `__init_subclass_with_meta__` wouldn't be called on any subclass of a class that was defined as abstract.